ONSdigital/eq-author
# eq-365 Create a Survey

### What is the context of this PR?

[Trello card](https://trello.com/c/i2ZKCSTR/367-8-author-create-a-survey)

Adds functionality for authors to add a survey and subsequently associate questionnaires with that survey.

### How to review 

- `npm install` new dependencies
- `./scripts/run_app.sh` to run the app (this not compiles assets too)
- login
- click 'Add survey' button
- Enter a title/id and submit
- Click "kebab" menu next to new survey
- Select "add" to add a questionnaire
- Enter a title and submit
- Confirm questionnaire has been added to survey
- Click title of questionnaire to edit
- Edit schema and click "save"
- Click "kebab" menu next to questionnaire and select 'delete' to remove the questionnaire
- Confirm questionnaire is removed from list.


### Who worked on the PR

@hamishtaplin
